InfernalRobotics/InfernalRobotics/Control/IPresetable.cs

```diff
@@ -11,7 +11,13 @@ public interface IPresetable
         void MoveTo(int presetIndex);
         void RemoveAt(int presetIndex);
         void Sort(IComparer<float> sorter = null);
-
+        /// <summary>
```

Comment:
this is a great idea, we should go through the rest of these interfaces and doc them up :)